scripts/quick-helm-install.sh

```diff
@@ -1,15 +1,9 @@
 #!/usr/bin/env bash
-set -euo pipefail
+set -eo pipefail
```

Comment:
Why not `-u` any more? I can't spot usage of uninitialized variable in the script. Which one is this, could you point me to the line that uses a variable before it is set?

Reply:
It's the `installflags`, bash complains that `${installflags[@]}` is an unbounded variable when -s is not set. I tried solving this by using `${installflags[@]:-}` as well as a few other patterns taken from [here](https://wiki.bash-hackers.org/syntax/pe), but didn't find a way to make it work for both the default way as well as the -s flag set.

Reply:
Rather than spend too much time finding the correct way to fix this with default values or expansions removing the `-u` was faster. Is there an important reason to keep `-u` after debugging is done and the script is finished?

Reply:
Hm. I cannot reproduce this. Let's investigate this a bit together, please.

I use the following `test.sh` script:
```sh
#!/usr/bin/env bash

set -euo pipefail

installflags=("$@")

echo "Hello, " \
	"${installflags[@]}"
```

Next, I run it with and without arguments and don't get any problems:
```sh
$ ./test.sh ; echo $?
Hello, 
0
$ ./test.sh Jan; echo $?
Hello,  Jan
0
```

Do you see the same?

Reply:
Please note that `installflags` is an array variable and the corresponding info is actually in this page https://wiki.bash-hackers.org/syntax/arrays

Reply:
I just ran `quick-helm-install.sh` script with `set ...` line modified to `set -euo pipefail` and without any arguments.
It worked without any issues.
Can you still reproduce the problem you had?

Reply:
I've just tried it out again and the result is as before. `./scripts/quick-helm-install.sh -s` works, but `./scripts/quick-helm-install.sh` without the `-s` fails with:
`./scripts/quick-helm-install.sh: line 69: installflags[@]: unbound variable`

Now if I instead assign a default `"${installflags[@]:-}"` the results are:
`./scripts/quick-helm-install.sh -s`: works
`./scripts/quick-helm-install.sh`: fails with `Error: INSTALLATION FAILED: expected at most two arguments, unexpected arguments: `

Using "" as a default value fails in the same way`"${installflags[@]:-""}"` 

Rather than doing too much trial and error with different ways of using default values I decided to turn off `set -u`. Any ideas on how to do this better?

Reply:
Why trial and error? The explanation of the root cause and the solution is just one google search away. https://stackoverflow.com/questions/7577052/bash-empty-array-expansion-with-set-u

Reply:
I had google searched too and didn't find a solution. I'll test yours and use it if it works.

Reply:
And it works, great. I'll be merging the branch now after committing this change.